### PR TITLE
Release Candidate.

### PR DESCRIPTION
**Pull request by randallli**
#### Don't forget:
- [ ] Identify the component the PR relates to in brackets in the title. ```[Buttons] Updated documentation```
- [ ] Link to GitHub issues it solves. ```closes #1234```
- [ ] Please add unit tests, snapshot tests, or manual testing steps depending on the change.
- [ ] Sign the CLA bot. You can do this once the pull request is opened.

[Contributing](https://github.com/material-components/material-components-ios/tree/develop/contributing) has more information and tips for a great
pull request.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,153 @@
+# 103.2.0
+
+Replace this text with a summarized description of this release's contents.
+
+## Breaking changes
+
+Replace this explanations for how to resolve the breaking changes.
+## New deprecations
+
+Replace this text with links to deprecation guides.
```
Comment (bryanoltman) [inline CHANGELOG.md:10]: Complete or delete these sections

File: .github/kokoro_label_all.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2020-present the Material Components for iOS authors. All Rights Reserved.
```
Comment (bryanoltman) [inline .github/kokoro_label_all.yml:1]: Should this file be in here?

File: .github/workflows/kokoro_labeler.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Copyright 2020-present the Material Components for iOS authors. All Rights Reserved.
```
Comment (bryanoltman) [inline .github/workflows/kokoro_labeler.yml:1]: Same Q about this file

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,153 @@
...
+## New features
+
+AppBar added an API for changing the stack view offset. 
+Flexible header added an API for showing the hairline and its color.
+Availability macro for internal use.
+AlertControllers added an API for titleIconView.
+Bottom drawer added an API for sticky status bars and adjustment when content size changes.
+Slider added an API for allowAnimatedValueChanges and trackEndsAreRounded or trackEndsAreInset.
+
+## API changes
```
Comment (bryanoltman) [inline CHANGELOG.md:20]: This can probably be deleted

Comment (bryanoltman) [reply CHANGELOG.md:20]: JK, components/Buttons/src/ButtonThemer/MDCFloatingActionButtonThemer.h was deleted

File: .github/kokoro_label_all.yml (hunk 2 above)
Comment (randallli) [reply .github/kokoro_label_all.yml:1]: yea its for copybara-service worker.
https://github.com/material-components/material-components-ios/pull/9503

File: .github/workflows/kokoro_labeler.yml (hunk 3 above)
Comment (randallli) [reply .github/workflows/kokoro_labeler.yml:1]: yep for copybara

File: CHANGELOG.md (hunk 1 above)
Comment (yarneo) [reply CHANGELOG.md:10]: +1

**Review by yarneo — APPROVED** [review]
Approving to unblock, but please fill out the changelog with the relevant information about this new release.

**Pull request merged**